src/pages/Map.scss

```diff
@@ -0,0 +1,33 @@
+.mapWrapper {
+  padding: 40px 20px;
+  background-color: var(--tangerine);
+  height: calc(100vh - 100px);
+}
+
+.google-map {
+  height: calc(100vh - 100px);
+  width: 90%;
+}
+
+.map {
+  text-align: center;
+  margin: 0 auto;
+  height: 90%;
+  width: 90%;
+  padding: 10px;
+  background-color: white;
+  border-radius: 10px;
+
+  div {
```

Comment:
If you move this out so it's its own selector `.map div` you can back out of this file being SASS